backend/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "backend",
+	"private": "true",
+	"engines": {
+		"node": ">=16 <17",
+		"npm": ">=8 <9"
+	},
+	"scripts": {
+		"migrate": "knex migrate:latest",
```

Comment:
Take out the common part `"knex": "node -r tsconfig-paths/register node_modules/knex/bin/cli.js",` and reuse into the next scripts

Reply:
Can't figure out how to split out repeated part from scripts command and pass it to all script, if this repeated part is not a separate command, but a part of single script (mb can move it to config-vars , but then it will lead to troubles with different syntax for accessing this var on different systems)

Reply:
Like this:
```json
"knex": "node -r tsconfig-paths/register node_modules/knex/bin/cli.js",
"migrate:dev": "npm run knex migrate:latest",
"migrate:make": "npm run knex migrate:make -- -x ts",
"migrate:down": "npm run knex migrate:down",
"migrate:rollback": "npm run knex migrate:rollback --all",
```

Reply:
Then second parts of command would be recognized as an argument,
IE `npm run knex migrate:make -- -x ts some_migration_name` won't understand `some_migration_name` as an migration-name argument, it will make a name of migration as `{timestampt}_ts` (although its migration file type arg) and skip migration_name

Reply:
Hmmmm, I didn't come across that the migration name was being skipped